protos/sources/data-node/api/v2/trading_data.proto

```diff
@@ -774,17 +774,21 @@ message AggregatedLedgerEntry {
     // The asset ID
     optional string asset_id = 5;
     // The type of account sent from
-    vega.AccountType sender_account_type = 6;
+    vega.AccountType from_account_account_type = 6;
```

Comment:
why the duplicity? from_account_account_type rather than from_account_type and to_account_type?

Reply:
i was not sure about from_account_type
But right okay, I'll change it